crates/fs/src/fs.rs

```diff
@@ -267,6 +279,36 @@ impl Fs for RealFs {
         })))
     }
 
+    // implement file watching using the notify crate
```

Comment:
This comment doesn’t seem necessary.

Reply:
I realized that hardly any function in Zed is documented or commented on. Is that the code styling you're going after? 

If so I can remove the comment so that it fits in with the rest of the codebase.

```suggestion
```

Reply:
Rustdoc comments on public items are helpful, but we try to keep other kinds of comments to a minimum, unless they're in a part of code that is otherwise unclear (and in those cases we try to make the code clearer instead of adding a comment).

Adding it to the contribution guide is a good idea 👍🏻